packages/wonder-blocks-modal/components/modal-backdrop.test.js

```diff
@@ -16,6 +16,19 @@ const exampleModal = (
     />
 );
 
+const exampleModalWithInitialFocus = (
+    <OnePaneDialog
+        content={
+            <div data-modal-content>
+                <input type="text" />
+                <button data-initial-focus />
+            </div>
+        }
+        title="Title"
+        footer={<div data-modal-footer />}
+    />
+);
+
 const exampleModalWithButtons = (
```

Comment:
It think it's preferable to inline these snippets even if it adds some duplicate code.  It makes the tests easier to read.  Also, we try to avoid DRY-ing out test code.

Reply:
Good point, thanks for clarifying on this :)